daemon/mgr/container.go

```diff
@@ -1016,7 +1016,7 @@ func (mgr *ContainerManager) updateContainerResources(c *Container, resources ty
 	if resources.CPUPeriod != 0 {
 		cResources.CPUPeriod = resources.CPUPeriod
 	}
-	if resources.CPUQuota != 0 {
+	if resources.CPUQuota != -1 {
```

Comment:
how about we change the compare to `resources.CPUQuota > 0`, the default value of cpu.cfs_quota_us is -1, and it accept value which equal -1 or greater than 1000, but we know -1 is not a useful value for quota, so we just check it should greater than 0.

Reply:
make sense !